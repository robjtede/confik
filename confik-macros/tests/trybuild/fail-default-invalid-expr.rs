#[derive(confik::Configuration)]
struct Config {
    #[confik(default = Hello World)]
    _param: String,
}

fn main() {}
