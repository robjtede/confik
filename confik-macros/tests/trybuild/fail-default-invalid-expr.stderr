error: expected `,`
 --> tests/trybuild/fail-default-invalid-expr.rs:3:30
  |
3 |     #[confik(default = Hello World)]
  |                              ^^^^^
